DigitalOcean.API/Models/Responses/TagResourcesStatistics.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalOcean.API.Models.Responses {
+    public class TagResourcesStatistics : TagResourceStatistics {
+        public TagResourceStatistics Droplets { get; set; }
+
+        public TagResourceStatistics Images { get; set; }
+
+        public TagResourceStatistics Volumes { get; set; }
+
+        public TagResourceStatistics VolumeSnapshots { get; set; }
+
+        public TagResourceStatistics Databases { get; set; }
```

Comment:
Intermediate class has a totals for Tag, then multiple properties for each explicit type,